Add inline documentation (#90)
nethesis/nethvoice-report: ui/src/doc-inline/en/graphs_area_1_pie_province.md

# Distribution of Incoming Calls by Province

Number of incoming calls divided by *Province* for the selected queues.

> The pie chart shows the total incoming calls by dividing it by
> origin of the calling number by dividing it at the level of *Province*, 
> for the queues select and compare the data proportionally.

Reviewer: ```suggestion
The pie chart shows the total incoming calls grouped by caller *Province*.
```
